src/assisted_test_infra/test_infra/helper_classes/day2_cluster.py

```diff
@@ -46,6 +46,19 @@ def _create(self) -> str:
     def update_existing(self) -> str:
         raise NotImplementedError("Creating Day2Cluster object from an existing cluster is not implemented.")
 
+    def set_hostnames_and_roles(self):
```

Comment:
Why is it different than `set_hostnames_and_roles` on `BaseCluster`?

Reply:
@eliorerz  .... let me know if you agree

Reply:
ack , will fix it on the parent